docs/build_RTD.sh

```diff
@@ -0,0 +1 @@
+python -m sphinx -T -E -b html -d _build/doctrees -D language=en . _build/html
```

Comment:
Fixes the shellcheck error about not knowing what the shell syntax to lint against.

```suggestion
#!/bin/sh
python -m sphinx -T -E -b html -d _build/doctrees -D language=en . _build/html
```

Reply:
After this precommit fix is made the PR can be merged. It could also be /bin/bash instead.